cirq-google/cirq_google/ops/z_gate_family.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Union
+
+import cirq
+from cirq.ops import raw_types
+from cirq_google.ops.physical_z_tag import PhysicalZTag
+
+
+_PHYSICAL_Z_TAG = PhysicalZTag()
+
+
+class ZGateFamily(cirq.GateFamily):
+    """A GateFamily which accepts either virtual or physical Z gates from Google devices.
+
+    If `physical_z` is true, accepts physical Z gates, i.e. `TaggedOperations` containing a Z gate
```

Comment:
This docstring has a confusing wording that makes the intended functionality ambiguous.  If `physical_z` is true, then this also should accept virtual Z gates.  The description makes it seem like accepting virtual and physical Z are mutually exclusive.

Reply:
+1

Reply:
If we keep this class, we should make it clear that you need two instances to support both in the comments.